src/background/services/paymentSession.ts

```diff
@@ -149,35 +147,15 @@ export class PaymentSession {
     await sleep(waitTime)
 
     while (this.active) {
-      try {
-        outgoingPayment = await this.openPaymentsService.createOutgoingPayment({
+      this.openPaymentsService
```

Comment:
How about we move this into a private function and avoid the `.then()`? Will be clearer IMO:
```ts
while (this.active) {
  void this.foo(...)
  await sleep(...)
}
```